htdocs/wp-content/install.php

```diff
@@ -34,16 +34,19 @@ function wp_install( $blog_title, $user_name, $user_email, $public, $deprecated
   populate_options();
   populate_roles();
 
-  if ( $language ) {
+/*  if ( $language ) {
```

Comment:
This completely removes the locale setting even in cases where we have it. Would it not be better to simply set the locale when there is one, and otherwise fall back to en_US?